src/views/Dashboard.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import DashboardHeader from '../components/DashboardHeader';
+import SideBar from '../components/sidebar/SideBar';
 
 const Dashboard: React.FunctionComponent = () => {
     return (
-        <React.Fragment>
-            <h1>Dashboard</h1>
-            <DashboardHeader />
-        </React.Fragment>
+        <div className="container view">
+            <React.Fragment>
+                <div className="columns">
+                    <div className="column is-one-quarter">
```

Comment:
With the section changes this looks a little wide, I would probably suggest `is-2` instead of `is-one-quarter`